### PR TITLE
[DVCSMP-2811] Update ranges for centralite battery values

### PR DESCRIPTION
**Pull request by varzac**
This just updates the range to be more conservative as well as match the
battery curve better.

@tpmanley @workingmonk

### CODE REVIEW

**Review by tpmanley — APPROVED** [review]
Changes look good. Needs a ticket and we should point at staging in order to get this out next week.

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy [hunk 1]
```diff
@@ -171,8 +171,8 @@ private Map getBatteryResult(rawValue) {
 			def pct = batteryMap[volts]
 			result.value = pct
 		} else {
-			def minVolts = 2.1
-			def maxVolts = 3.0
+			def minVolts = 2.4
+			def maxVolts = 2.7
```
Comment (workingmonk) [inline devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy:175]: battery values would be 100, 67, 33 and 0. is that ok?

**Comment by workingmonk** [comment]
can you reach out to  Steven for testing? this might need a v1 hub to avoid local DTH overriding your change. Matt, has a h/w battery simulator that Steven can use

**Comment by vseven** [comment]
https://community.smartthings.com/t/st-multi-sensor-battery-reporting-issues/98924

This broke many sensors and should be reverted or fixed properly.

**Pull request merged**